scapy/layers/tls/tools.py

```diff
@@ -176,7 +177,7 @@ def _tls_aead_auth_encrypt(alg, p, write_seq_num):
     write_seq_num = struct.pack("!Q", write_seq_num)
```

Comment:
It’s not related to your PR, but I just noticed it. Would you mind replacing the str (line above this comment)
by a `raw()` call (import if needed from scapy.compat).
Would be real nice

Reply:
Done.